daisymfcgui/pom.xml

```diff
@@ -0,0 +1,167 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.daisy.pipeline.lite</groupId>
+    <artifactId>org.daisy.pipeline.lite.libs</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>DAISY pipeline 1 lite libraries aggregator for the word addin</name>
+    <packaging>pom</packaging>
+    <properties>
+        <osgi.platform>win32.win32.x86</osgi.platform>
+    </properties>
+
+    <repositories>
+        <repository>
+            <id>local-repo</id>
+            <url>file:///${project.basedir}/local-repo</url>
+        </repository>
+    </repositories>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.eclipse.platform</groupId>
+                <artifactId>org.eclipse.swt</artifactId>
+                <version>3.108.0</version>
+            </dependency>
+            <dependency>
+                <groupId>org.eclipse.platform</groupId>
+                <artifactId>org.eclipse.swt.win32.win32.x86</artifactId>
+                <version>3.108.0</version>
+            </dependency>
+            <dependency>
+                <groupId>org.eclipse.platform</groupId>
+                <artifactId>org.eclipse.swt.win32.win32.x86_64</artifactId>
+                <version>3.108.0</version>
+            </dependency>
+            <dependency>
+                <groupId>org.eclipse.platform</groupId>
+                <artifactId>org.eclipse.jface</artifactId>
+                <version>3.14.100</version>
+            </dependency>
+            <dependency>
+                <groupId>org.eclipse.platform</groupId>
+                <artifactId>org.eclipse.core.commands</artifactId>
+                <version>3.9.200</version>
+            </dependency>
+            <dependency>
+                <groupId>org.eclipse.platform</groupId>
+                <artifactId>org.eclipse.equinox.common</artifactId>
+                <version>3.10.100</version>
+            </dependency>
+            <dependency>
+                <groupId>javax.xml.stream</groupId>
+                <artifactId>stax-api</artifactId>
+                <version>1.0-2</version>
+            </dependency>
+            <!-- Needs to be within modules -->
+            <!-- <dependency> <groupId>org.daisy.pipeline</groupId> <artifactId>org.daisy.pipeline</artifactId> 
+				<version>1.0.12</version> <scope>system</scope> <systemPath>${project.basedir}/../../dist/pipeline.jar</systemPath> 
+				</dependency> <dependency> <groupId>org.daisy.pipeline</groupId> <artifactId>org.daisy.util</artifactId> 
+				<version>1.0.12</version> <scope>system</scope> <systemPath>${project.basedir}/../../dist/org.daisy.util.jar</systemPath> 
+				</dependency> -->
```

Comment:
What does this comment mean?

Reply:
Fixed when Nico fixed up commit fba71f39fd7f4487978529b8c5bbd76730aa62f2 to 73691508.